### PR TITLE
'Dogecoin has No Developers' Article

### PR DESCRIPTION
**Pull request by RichieJohn**
Link will need checking. Assuming that this will be the link for the Foundation website. As always, please edit wherever. Made a few tweaks - but still v much in line with original content.

### CODE REVIEW

File: content/dogepedia/faq/dogecoin-has-no-developers.md [hunk 1]
```diff
@@ -5,9 +5,10 @@ date = "2021-10-22"
...
+Oh, yes it does - and it always has!  
 
+Dogecoin’s primary purpose has always been to be used as a currency, and since it was forked from Litecoin and then reforked from Bitcoin — with changed parameters to make it more suitable as a cryptocurrency — Dogecoin inherited already feature-complete code bases. So, for many years, instead of having new shiny features, the development team worked mostly as maintainers of what was already there. Not once, in its history, Dogecoin did *not* have a group of developers working on it. 
 
+In the future, the plan is to focus more efforts and resources to enhance Dogecoin so that it fulfills its aim to become a currency of the People. This will require a combination of part-time contributors, volunteers, but also full-time staff including developers and engineers.   
 
+The Dogecoin Foundation’s [Trailmap](https://foundation.dogecoin.com/trailmap/) lists various projects as the ecosystem works towards its shared aim of becoming a currency for the People, by the People! This Trailmap is not authoritative, but an indication of where to go that is open for community feedback.
```
Comment (MishaBoar) [inline content/dogepedia/faq/dogecoin-has-no-developers.md:14]:
```suggestion
The Dogecoin Foundation’s [Trailmap](https://foundation.dogecoin.com/trailmap/) lists various projects as the ecosystem works towards its shared aim of becoming a currency for the People, by the People! This Trailmap is not authoritative, but an indication of where to go that is open for community feedback.

Even though a lot of actual work is done outside of this repository, some of the development efforts can be followed on Github in the [**Dogecoin Core repository**](https://github.com/dogecoin/dogecoin). [**r/dogecoindev**](https://reddit.com/r/dogecoindev) on reddit is another good channel to discuss everything around Dogecoin's development and interact with the developers.
```

**Review by MishaBoar — CHANGES_REQUESTED** [review]
Great! Just added some final linking to current repository, which we can always update on the fly later.

File: content/dogepedia/faq/dogecoin-has-no-developers.md [hunk 2]
```diff
@@ -5,9 +5,10 @@ date = "2021-10-22"
...
-
-In the future, the plan is to focus more efforts and resources to enhance Dogecoin, and make it fullfill its aim to become a crypto of the people also in its technical implementation. This will require a combination of part time contributors, volunteers, but also full time developers and engineers. Alongside, the foundation and the advisors will provide a "roadmap" to aim at possible goals: roadmap that will never be authoritative or normative, but an indication of where to go.
+Oh, yes it does - and it always has!  
 
+Dogecoin’s primary purpose has always been to be used as a currency, and since it was forked from Litecoin and then reforked from Bitcoin — with changed parameters to make it more suitable as a cryptocurrency — Dogecoin inherited already feature-complete code bases. So, for many years, instead of having new shiny features, the development team worked mostly as maintainers of what was already there. Not once, in its history, Dogecoin did *not* have a group of developers working on it.
```
Comment (MishaBoar) [inline content/dogepedia/faq/dogecoin-has-no-developers.md:10]:
```suggestion
Dogecoin’s primary purpose has always been to be used as a currency. Dogecoin was originally a fork of a cryptocurrency called Luckycoin - itself a fork of Litecoin. Billy Markus, co-creator of Dogecoin with Jackson Palmer, acted as a solo developer for the first Dogecoin Core releases. In 2014, Billy and Jackson left Dogecoin development, and a new development team was formed.  Over the years, the Dogecoin development team has consisted of a small group of maintainers coordinating the work of more than 40 contributors.

In 2015, starting from Dogecoin Core 1.10.0, the Dogecoin development team decided to re-fork Dogecoin off Bitcoin, re-applying Litecoin changes on top of the Bitcoin codebase. Thus, throughout its history, Dogecoin inherited already feature-complete code bases, and it has shared a good part of its identity with Bitcoin. Its uniqueness came from changed parameters, optimizations, and parts of the code that were not inherited from Litecoin or Bitcoin. These changes were always aimed at making Dogecoin a viable currency for day to day spending, with a focus on utility and ease of use.
```

Comment (MishaBoar) [reply content/dogepedia/faq/dogecoin-has-no-developers.md:10]: The original paragraph inherited from my stub contained some imprecise statements. I fixed it and expanded it in my suggestion.

**Review by MishaBoar — CHANGES_REQUESTED** [review]

**Review by MishaBoar — APPROVED** [review]

**Review by RichieJohn — COMMENTED** [review]
Looks good! This needs to be approved again before merging, I think, @MishaBoar :)

**Comment by RichieJohn** [comment]
Oh - looks like it's automagically merged. :)

**Comment by RichieJohn** [comment]
Ah - looks like it's already merged.

**Pull request merged**